Sources/VLCEditController.swift

```diff
@@ -93,10 +100,11 @@ private extension VLCEditController {
 // MARK: - VLCEditToolbarDelegate
 
 extension VLCEditController: VLCEditToolbarDelegate {
-    func editToolbarDidAddToPlaylist(_ editToolbar: VLCEditToolbar) {
+    func addToNewPlaylist() {
         let alertInfo = TextFieldAlertInfo(alertTitle: NSLocalizedString("PLAYLISTS", comment: ""),
                                            placeHolder: "NEW_PLAYLIST")
```

Comment:
placeHolder is not localized, I know not your code :)

Reply:
I will add separate commit for this.